apps/src/blockly/utils.ts

```diff
@@ -312,3 +312,39 @@ export function getAllBlocks(): BlockList {
       : []),
   ];
 }
+
+export function disableOrphanFunctionCalls(
```

Comment:
Can this just be called `disableOrphans`? The bulk of the work in this function is `updateBlockEnabled`, which is performed on every top block, not just functions. The purpose is to disable any block that has been disconnected.